a/anicet.md

```diff
@@ -0,0 +1,10 @@
+# Anicet
+
+
+![Anicet](anicet.JPG)
+
+This singer doesn't have biography but you can go to youtube if need more video of him.
+
+**Genre:** slow
+
+**Artist homepage:** [Anicet](http://www.youtube.com)
```

Comment:
This link could be more helpful. Is there a channel for this artist on Youtube?

Reply:
Yes,there is.

Reply:
Then link the channel!